### PR TITLE
Package for the resultsFile Python module

### PR DESCRIPTION
**Pull request by scemama**
Python module to read output files of quantum chemistry programs.

It depends on py-setuptools only at build time, but I get the following error:
```
  error: option --single-version-externally-managed not recognized
```
So I fixed it by setting the type to 'run', which is a bit dirty.

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-resultsfile/package.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    """Python module to read output files of quantum chemistry programs"""
+
+    homepage = "https://gitlab.com/scemama/resultsFile"
+    url      = "https://gitlab.com/scemama/resultsFile/-/archive/v1.0/resultsFile-v1.0.tar.gz"
+
+    maintainers = ['scemama']
+
+    version('1.0', sha256='e029054b2727131da9684fa2ec9fb8b6a3225dc7f648216a9390267b2d5d60c3')
+
+    depends_on('python@2.7:2.8.999', type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-resultsfile/package.py]: It doesn't work with Python 3?

File: var/spack/repos/builtin/packages/py-resultsfile/package.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+    # It depends on py-setuptools only at build time, but I get the following
+    # error:
+    #   error: option --single-version-externally-managed not recognized
+    # So I fixed it by setting the type to 'run' here, which is a bit dirty.
+    depends_on('py-setuptools', type='run')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-resultsfile/package.py]: I don't see anywhere where setuptools is used at all, only distutils. I think you can delete this dependency.

Comment (scemama) [reply var/spack/repos/builtin/packages/py-resultsfile/package.py]: The setup.py script has "import setuptools" at the beginning of the script

File: var/spack/repos/builtin/packages/py-resultsfile/package.py (hunk 1 above)
Comment (scemama) [reply var/spack/repos/builtin/packages/py-resultsfile/package.py]: Version 1.0 is for Python2 and version 2.0 is for Python3.
I updated the package file to provide for v1.0 for Python2 or v2.0 for Python3.

File: var/spack/repos/builtin/packages/py-resultsfile/package.py (hunk 2 above)
Comment (scemama) [reply var/spack/repos/builtin/packages/py-resultsfile/package.py]: You are right, py-setuptool is not needed. I removed it. Thanks!

**Pull request merged**